src/WinUI/ArcGISRuntime.WinUI.Viewer/SamplePage.xaml.cs

```diff
@@ -29,10 +37,18 @@ public SamplePage()
 
             // Get selected sample and set that as the DataContext.
             DataContext = SampleManager.Current.SelectedSample;
+            _selectedSample = SampleManager.Current.SelectedSample;
 
             // Load and show the sample.
             SampleContainer.Content = SampleManager.Current.SampleToControl(SampleManager.Current.SelectedSample);
 
+            if (ScreenshotManager.ScreenshotSettings.ScreenshotEnabled)
+            {
+                KeyDown += ScreenshotKeyDown_Event;
+                SampleContainer.Width = ScreenshotManager.ScreenshotSettings.Width.HasValue ? ScreenshotManager.ScreenshotSettings.Width.Value : double.NaN;
```

Comment:
Do we know what happens if we set the value to `double.nan`?

Reply:
It works as intended, it resets the container back to the default size.